core/server/services/url/utils.js

```diff
@@ -17,10 +17,13 @@ const moment = require('moment-timezone'),
  */
 function getApiPath(options) {
     const apiVersions = config.get('api:versions');
-    let version = options.version || 'deprecated';
-    let type = options.type || 'content';
-    let versionData = apiVersions[version];
-    let versionPath = versionData[type];
+    let requestedVersion = options.version || 'deprecated';
+    let requestedVersionType = options.type || 'content';
+    let versionData = apiVersions[requestedVersion];
+    if (typeof versionData === 'string') {
+        versionData = apiVersions[versionData];
+    }
+    let versionPath = versionData[requestedVersionType];
```

Comment:
Could you pls add some unit tests to see how this improvement will be used in the code base?
Thanks :)

Reply:
Yeah sure 👍

Reply:
@kirrg001 Updated tests [here](https://github.com/TryGhost/Ghost/pull/10012/files#diff-46bc90683d4ecb212a93939d49576e5eR441) to cover this, plus updated use [here](https://github.com/TryGhost/Ghost/pull/10012/files#diff-66fad1e189e44d0801f787cb1d1c989bR41) as usecase for this change.